build/ci.go

```diff
@@ -122,6 +122,15 @@ var (
 	// Note: wily is unsupported because it was officially deprecated on lanchpad.
 	// Note: yakkety is unsupported because it was officially deprecated on lanchpad.
 	debDistros = []string{"trusty", "xenial", "zesty", "artful"}
+
+	// stable is a list of packages known to be stable to do full test, list and cyclo
```

Comment:
comment mentions `stable` instead of `stablePackages`

Reply:
:+1: